src/mod_ping.erl

```diff
@@ -140,6 +151,7 @@ handle_remote_hook(HandlerState, mod_ping, Args, C2SState) ->
     handle_remote_call(Args,
                        ejabberd_c2s_state:jid(C2SState),
                        ejabberd_c2s_state:server(C2SState),
```

Comment:
don't see any difference.

Reply:
One argument less, and less ignoring below. Only `route_ping_iq/3` is using the server here, and it's also taking the JID, so it can extract the server from it. Just an opinionated one.